src/gui/torrentcontentmodelitem.cpp

```diff
@@ -74,9 +74,7 @@ qreal TorrentContentModelItem::progress() const
 {
     Q_ASSERT(!isRootItem());
 
-    if (m_size > 0) return m_progress;
-
-    return 1;
+    return (m_size > 0) ? m_progress : 1;
```

Comment:
But why it returns 1 in case of `m_size == 0`?

Reply:
When you add a torrent job and point to existing files on disk, if the file size == 0 then libtorrent reports progress = 0.

Reply:
>When you add a torrent job and point to existing files on disk, if the file size == 0

Is this the case when the torrent contains really empty files?

Reply:
>Is this the case when the torrent contains really empty files?

Yes.